package-overrides/github/christopherthielen/ui-router-extras@0.0.14.json

```diff
@@ -0,0 +1,15 @@
+{
+  "files": [
```

Comment:
Let's leave this out, in case usage of the modular versions is useful for others in future.